src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -659,22 +928,18 @@ end
 		amount = 100
 	})
 	```
-	
+
 	@param data ResourceEvent
-	@return { [any]: any }
+	@return Promise
```

Comment:
```suggestion
	@return Promise<T>
```
Unless the type is already defined, you need to define custom return types with Moonwave by denoting them with a after the type (Eg. `Promise<T>`)